sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloInstanceTest.java

```diff
@@ -67,12 +78,62 @@ public void assertGet() {
     @Test
     @SneakyThrows
     public void assertWatch() {
+        assertWatchUpdateChangedType("/test/children/1", "value3");
+    }
+    
+    @Test
+    @SneakyThrows
+    public void assertGetWithNonExistentKey() {
+        assertNull(configCenterRepository.get("/test/nonExistentKey"));
+    }
+    
+    @Test
+    @SneakyThrows
+    public void assertUpdate() {
+        assertWatchUpdateChangedType("/test/children/1", "newValue1");
+        assertThat(configCenterRepository.get("/test/children/1"), is("newValue1"));
+    }
+    
+    @Test
+    @SneakyThrows
+    public void assertWatchDeletedChangedType() {
         final SettableFuture<DataChangedEvent> future = SettableFuture.create();
-        configCenterRepository.watch("/test/children/1", future::set);
+        configCenterRepository.watch("/test/children/1", new DataChangedEventListener() {
+    
+            @Override
+            public void onChange(final DataChangedEvent dataChangedEvent) {
+                future.set(dataChangedEvent);
+            }
+        });
+        embeddedApollo.deleteProperty("orchestration", "test.children.1");
         embeddedApollo.addOrModifyProperty("orchestration", "test.children.1", "value3");
         DataChangedEvent changeEvent = future.get(5, TimeUnit.SECONDS);
         assertThat(changeEvent.getKey(), is("/test/children/1"));
-        assertThat(changeEvent.getValue(), is("value3"));
+        assertNull(changeEvent.getValue());
+        assertThat(changeEvent.getChangedType(), is(DataChangedEvent.ChangedType.DELETED));
+        assertNull(configCenterRepository.get("/test/children/1"));
+    }
+    
+    @Test
+    @SneakyThrows
+    public void assertWatchAddChangedType() {
+        assertWatchUpdateChangedType("/test/children/newKey", "newVaule");
```

Comment:
It's not a good idea for using `assertWatchUpdateChangedType ` method  at `assertWatchAddChangedType ` method, maybe we should rename these methods and make them more meaningful.